lib/ui/views/home/home_viewmodel.dart

```diff
@@ -194,7 +193,7 @@ class HomeViewModel extends BaseViewModel {
                     ),
                     const SizedBox(width: 8.0),
                     Text(
-                      'v$_managerVersion',
+                      'v$_latestManagerVersion',
```

Comment:
Get this value from _managerAPI.getLatestManagerVersion() method

Reply:
what do you mean by `_managerAPI.getLatestManagerVersion()`?
there is no method by that name

Reply:
There is, it was used on line 118 but you removed it.

Reply:
i removed it in favor of the `_latestManagerVersion` variable that calls the `_managerAPI.getLatestManagerVersion()` method in initialize

Reply:
the variable in line 118 was only within the scope of that function, i didnt want to do another api request since alex will bonk me.
i deleted it and made a variable on init instead so it calls for it once instead.
the `managerVersion` variable was used only for the text previously anyways, so i went ahead and deleted that too since the text would be calling from the new variable